### ISSUE TITLE
ArcTangentToArc

### ISSUE THREAD

**Issue by anpoit00** [Macad3D/Macad3D#46]
I am not sure, but the condition cannot be selected
`ArcTangentToArc = constraintType::arcTangentToArc, `

https://user-images.githubusercontent.com/5604859/220067096-f9aa92a9-9013-4249-bb3c-184eee211c24.mp4

**Comment by dipts** [comment]
That's right. The collection in `SketchSolve::ConstraintType` are _micro constraints_ which are used in the solver. The sketch constraints configure one or more of these micro constraints to produce the desired behavior. Since the segments in Sketcher differ in their definition to the basic elements of the solver, there is no one-to-one correspondence.

To enable a tangency constraint between two arcs this has to be implemented in `SketchConstraintTangent` and the required micro constraints have to be configured there. A nice beginner task.  :wink:
